mappings/net/minecraft/item/VanishableArmorItem.mapping

```diff
@@ -0,0 +1,8 @@
+CLASS net/minecraft/unmapped/C_kelbxmwg net/minecraft/item/VanishableArmorItem
```

Comment:
actually one thing.
this seems like *way* too specific of a name, given that all equipment that can be worn can have vanishing curse
could we not just name this something like `Equippable` or `Equipment`?
probably the longest name I'd be ok with is `WearableEquipment`

Reply:
I think this used to be `Vanishable`

Reply:
No, but this *extends* `Vanishable` and is only used by ArmorItems, Skulls, Carved Pumpkins, Elytras and Shields.

Reply:
Ah ok. Then i would say `WearableVanishable` or the other way around

Reply:
I hate that so much though

Reply:
yeah no that's a bad name
I checked mojang and they simply call it `Equippable` so I think it's best we match that

Reply:
sure!